src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -11,6 +16,12 @@ namespace System.Data.SqlClient.SNI
     /// </summary>
     internal class SNIProxy
     {
+        private static readonly char[] s_semicolonSeparator = { ';' };
+        private static readonly char[] s_commaSeparator = { ',' };
+        private static readonly char[] s_backSlashSeparator = { '\\' };
+        private static readonly int SQL_SVR_BROWSER_PORT = 1434;
+        private static readonly int DEFAULT_SQL_SVR_PORT = 1433;
```

Comment:
Nit: The constants can be `const` instead of `static readonly`. Also, the [coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md) for CoreFX is to use PascalCasing for constants:

> We use PascalCasing to name all our constant local variables and fields. The only exception is for interop code where the constant value should exactly match the name and value of the code you are calling via interop.

So something like:

```c#
private const int SqlServerBrowserPort = 1434;
private const int DefaultSqlServerPort = 1433;
```